.circleci/config.yml

```diff
@@ -67,14 +67,7 @@ jobs:
     environment:
       - FEDERATION: "true"
       - NODE_ENV: "test"
-      - ACL_SERVICE_URL: "https://acl-intb.ciscospark.com/acl/api/v1"
-      - ATLAS_SERVICE_URL: "https://atlas-intb.ciscospark.com/admin/api/v1"
-      - CONVERSATION_SERVICE: "https://conversation-intb.ciscospark.com/conversation/api/v1"
-      - ENCRYPTION_SERVICE_URL: "https://encryption-intb.ciscospark.com/encryption/api/v1"
-      - IDBROKER_BASE_URL: "https://idbrokerbts.webex.com"
```

Comment:
Test users still needs `IDBROKER_BASE_URL` and `CONVERSATION_SERVICE` to create them on integration